eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -226,7 +231,8 @@ jobs:
   - template: /eng/common/TestResources/deploy-test-resources.yml
     parameters:
       ServiceDirectory: ${{ parameters.ServiceDirectory }}
-      Location: ${{ parameters.Location }}
+      Location: "$(KVLocation)"
+      ArmTemplateParameters: "@{ enableHsm = [System.Convert]::ToBoolean($(EnableHSM)) }"
```

Comment:
Is this working? It would need to either generate `"true"` or `$true` for it to be valid powershell object literal syntax.

Reply:
it works, this way it oconverts to wither true or false based on what  is defined in the matrix

Reply:
it converts to $true or $false,